rainforest-cli.go

```diff
@@ -26,8 +23,16 @@ var (
 	// Rainforest API client
 	api *rainforest.Client
 
-	// default output for printing
-	out io.Writer = os.Stdout
+	// default output for printing resource tables
+	tablesOut io.Writer = os.Stdout
+
+	// Run status polling interval
+	runStatusPollInterval = time.Second * 5
+
+	// Batch size (number of rows) for tabular var upload
+	tabularBatchSize = 50
+	// Concurent connections when uploading CSV rows
+	tabularConcurency = 1
```

Comment:
It's more for us to have it in a easy accessible place if we decide to change it later on.